src/backend/common/CommandBuffer.cpp

```diff
@@ -238,9 +257,75 @@ namespace backend {
             return true;
         };
 
+        // TODO(kainino@chromium.org): Manage this state inside an object, change lambda to a method
+        RenderPassBase* currentRenderPass = nullptr;
+        FramebufferBase* currentFramebuffer = nullptr;
+        uint32_t currentSubpass = 0;
+        auto beginSubpass = [&]() -> bool {
+            auto& subpassInfo = currentRenderPass->GetSubpassInfo(currentSubpass);
+            for (auto& kv : subpassInfo.colorAttachments) {
+                if (auto* tv = currentFramebuffer->GetTextureView(kv.second)) {
```

Comment:
Yep realized that afterwards, but forgot to edit this comment, TODO sounds good.